business/iter8.go

```diff
@@ -0,0 +1,279 @@
+package business
+
+import (
+	"encoding/json"
+	"github.com/kiali/kiali/kubernetes"
+	"github.com/kiali/kiali/models"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	kube "k8s.io/client-go/kubernetes"
+	"github.com/kiali/kiali/config"
+	"k8s.io/client-go/rest"
+	"time"
+	"fmt"
+	"github.com/kiali/kiali/log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	iter8v1alpha1 "github.com/iter8-tools/iter8-controller/pkg/apis/iter8/v1alpha1"
```

Comment:
Also, as commented, this import may have implications to also add the whole iter8-controller as a dependency.
I wonder if we can just clone the structs needed as we did in Istio too, I'll tackle that too.

Reply:
I notice that all the dependencies package were also checkin to git. I am not sure if I should do that or not.